CHANGELOG.md

```diff
@@ -7,19 +7,47 @@ next release (yyyy-mm-dd)
 
 #### ⛔ Breaking Changes
 
+#### New Features
+
+#### Bug fixes, Minor Improvements
+
+### UI Changes
+
+1.47.0 (2023-07-05)
+-------------------
+### Backend Changes
+
+#### ⛔ Breaking Changes
+
 * [SPM] Due to a breaking change in OpenTelemetry's prometheus exporter ([details](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.80.0))
   metric names will no longer be normalized by default, meaning that the expected metric names would be `calls` and
   `duration_*`. Backwards compatibility with older OpenTelemetry Collector versions can be achieved through the following flags:
   * `prometheus.query.normalize-calls`: If true, normalizes the "calls" metric name. e.g. "calls_total".
   * `prometheus.query.normalize-duration`: If true, normalizes the "duration" metric name to include the duration units. e.g. "duration_milliseconds_bucket".
 
-
 #### New Features
+* [Cassandra] Add Configuration.Close() to ensure TLS cert watcher is closed ([@kennyaz](https://github.com/kennyaz) in [#4515](https://github.com/jaegertracing/jaeger/pull/4515))
+* Add *.kerberos.disable-fast-negotiation option to Kafka consumer ([@pmuls99](https://github.com/pmuls99) in [#4520](https://github.com/jaegertracing/jaeger/pull/4520))
 
 #### Bug fixes, Minor Improvements
 
+* Support normalized metric names ([@albertteoh](https://github.com/albertteoh) in [#4555](https://github.com/jaegertracing/jaeger/pull/4555))
```

Comment:
this one does sound like new feature, you even mention it in breaking changes. I would make the caption more descriptive.